test/TestUtils/TestProject.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
```

Comment:
Will Clean up these test utils, moving `TestProject` and `TestProjectFixture` here.